app/src/organisms/ODD/QuickTransferFlow/utils/retrieveLiquidClassValues.ts

```diff
@@ -373,7 +381,9 @@ const getLiquidClassValues = (
     tipPositionAspirate: aspirate?.aspiratePosition.offset.z ?? 0,
     submergeAspirate: {
       speed: aspirate?.submerge.speed ?? 0,
-      positionFromBottom: aspirate?.submerge.startPosition.offset.z ?? 0,
+      position: aspirate?.submerge.startPosition.offset.z ?? 0,
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      positionReference: aspirate?.submerge.startPosition.positionReference!,
```

Comment:
couldn'r you do something like this to avoid needing to add all the esLint lines?
```suggestion
      positionReference: aspirate?.submerge.startPosition.positionReference ?? undefined
```